### DISCUSSION TITLE
Google keeps blocking site and impacts click and open tracking

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2541] [Help with using Postal]
This is the second time google has listed the postal server as dangerous. I already had it removed once before.
![image](https://github.com/postalserver/postal/assets/5223052/2f7ae489-822b-4bf0-bcda-fd4827ea3d27)

![image](https://github.com/postalserver/postal/assets/5223052/4d8669ad-27f3-40a7-9324-7356a5dd1ee9)
![image](https://github.com/postalserver/postal/assets/5223052/e5bf8c44-70b3-4905-929d-472241c9d6ad)

Any thoughts?

**Comment by willpower232** [comment]
Is it possible you're re using a domain that was dodgy in the past or resembles other domains which are dodgy?

Are you hosting other things on the same domain or IP address?

Not really a lot you can do without getting more details from Google about what is going wrong.
